stdlib/http/src/main/ballerina/http/http_secure_client.bal

```diff
@@ -28,6 +28,11 @@ public type AuthScheme "Basic"|"OAuth2"|"JWT";
 @final public AuthScheme OAUTH2 = "OAuth2";
 @final public AuthScheme JWT_AUTH = "JWT";
 
+public type CredentialBearer "AuthHeaderBearer"|"PostBodyBearer";
```

Comment:
Shall we document the type and variables?

Reply:
Added in commit 7666004